requirements.txt

```diff
@@ -1,2 +1,3 @@
 git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
+git+https://github.com/edx/XBlock.git@cdyer/scorable-mixin
```

Comment:
Is this just a temporary thing? Because normally we'd want to stick with the pattern of letting `xblock-utils` specify the version of `XBlock` that is being used here (for tests). So it would be best to bump the dependency in `xblock-utils` then bump the `xblock-utils` version here.